doc/index.rst

```diff
@@ -27,7 +27,7 @@ including MCNP5_, Geant4_, Fluka_, Tripoli4_, and Shift_. There are also efforts
 planned to integrate DAGMC into other codes such as MCNP6_, Serpent2_,
 OpenMC_, Phits_, and Frensie.
 
-DAGMC currently relies on using the solid modeling software Cubit_ or Trelis_ to
+DAGMC currently relies on using the solid modeling software Cubit_ (commercial name Trelis_) to
```

Comment:
That's possible. I just wanted to link them more closely than saying "Cubit or Trelis". Maybe something like "solid modeling software produces by csimsoft, Cubit or Trelis, ..."?